Source/Core/BoogiePL.atg

```diff
@@ -407,6 +409,7 @@ Consts<.out List<Variable>/*!*/ ds.>
   ]
   IdsType<out xs>
   [ OrderSpec<out ChildrenComplete, out Parents> ]
+  ( ";" | "uses" "{" { Axiom<out ax>(. axioms.Add(ax); .) } "}")
```

Comment:
This makes `uses` a reserved keyword in Boogie. Any previous program that used `uses` as an identifier will no longer parse. I think this is okay, but we should make sure to bump the Boogie version accordingly (to a new minor number, probably).

If there's concern about making `uses` a keyword, an alternative would be to add it as a _contextual keyword_. That's a little tricky here, because

```
function F(x: int): MyType uses {
  x + 5
}
```

could mean one thing with

    type MyType = int;

and something different with

```
type MyType T = int;
type uses;
```

Reply:
I bumped the minor version number